src/mono/mono/mini/simd-intrinsics.c

```diff
@@ -841,6 +841,11 @@ static SimdIntrinsic crypto_aes_methods [] = {
 	{SN_get_IsSupported}
 };
 
+static SimdIntrinsic neon_aes_methods [] = {
+	{SN_PolynomialMultiplyWideningLower, OP_XOP_X_X_X, SIMD_OP_ARM64_PMULL64_LOWER},
```

Comment:
I just realized that for some of these we are only specifying an enum value (SN_whatever), and for some of them we are specifying op codes.  Why the difference?

Reply:
It depends if it is necessary to have the opcode or not. For `SN_get_IsSupported`, all we needed to to was
```
if (id == SN_get_IsSupported) {
			EMIT_NEW_ICONST (cfg, ins, supported ? 1 : 0);
			return ins;
		}
```
For the methods with op codes that I have been working on, they went through this common logic:
https://github.com/dotnet/runtime/blob/master/src/mono/mono/mini/simd-intrinsics.c#L987-L1021